RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByIsolation.cc

```diff
@@ -195,10 +228,29 @@ class PFRecoTauDiscriminationByIsolation : public PFTauDiscriminationProducerBas
   double offsetRelativeSumPt_;
   double customIsoCone_;
   
+  bool applyPhotonPtSumOutsideSignalConeCut_;
+  double maxAbsPhotonSumPt_outsideSignalCone_;
+  double maxRelPhotonSumPt_outsideSignalCone_;
+
+  bool applyFootprintCorrection_;
+  struct footprintCorrectionType
```

Comment:
types should start with a capital letter (and instances of this type with a lower case) .. there's no need to add "Type" to the name this way.